make-debs.sh

```diff
@@ -33,7 +33,7 @@ git clean -dxf
 # c) compares higher than any previous commit
 # d) contains the short hash of the commit
 #
-vers=$(git describe --match "v*" | sed s/^v//)
+vers=$(git describe --match "v*" | sed 's/^v//')
```

Comment:
why do you think it was wrong without the quotes? could you update the commit message with the rationale?

Reply:
@tchaikov Please ignore this commit.
I hits the sed usage error 3 weeks ago when building ceph ubuntu packages. Now there's no error without this commit.